docs/pages/introduction/why-not-expo.mdx

```diff
@@ -4,107 +4,40 @@ sidebar_title: Limitations
-
-Apps built with `eas build -p ios|android` [include only your app's explicit native dependencies](https://blog.expo.dev/expo-managed-workflow-in-2021-d1c9b68aa10), and so this limitation does not apply if you use EAS Build. [Learn about how to use it](/build/introduction).
-
-</Collapsible>
-
-> We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
-
-## Limitations of the bare workflow
-
-In the bare workflow, we have full access to the underlying native projects and any native code. It's a "bare" native project with React Native and one or more packages from the Expo SDK installed. Anything that you can do in a native project is possible here.
-
-The following list is therefore specifically oriented towards the limitations that exist around using Expo tools and services in the bare workflow.
-
-<Collapsible summary={<span><span className="strike">Build service only works in the managed workflow</span> (✅ Resolved in December 2020)</span>}>
-
-~~To build your app binaries for distribution on the Apple App Store and Google Play Store you will need to follow the same steps that you would in any native project, the Expo build service can't handle it for you. We are working on bringing bare workflow support to the build service in the near future.~~
-
-You can now use [EAS Build](/build/introduction) to build and sign your apps just as easily as in the managed workflow! [Read the announcement blog post](https://blog.expo.dev/expo-application-services-eas-build-and-submit-fc1d1476aa2e).
-
-</Collapsible>
-
-<Collapsible summary="Configuration must be done on each native project rather than once with app.json">
-
-Configuring app icons, launch screen, and so on must be configured in the native projects for each platform using the standard native tooling, rather than once using a simple JSON object.
-
-</Collapsible>
+**All of the above have been addressed with EAS Build which is compatible with all type of React Native projects. If you come across them, you can transition your project to use [development builds](/development/introduction/).**
 
 ## Up next
 
-If you've been reading along each section of the introduction then you will have a pretty good high-level understanding of Expo tools.
+We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
```

Comment:
```suggestion
We are either actively working on or planning to build solutions to the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
```
Wanted to couch that statement a little bit, just because some things are limited by their very nature (e.g., Expo Go isn't including custom native code anytime soon)

It might be too much, but wondering if we name-drop Github Discussions as the place for more detailed RfC's/ feature proposals (https://github.com/expo/expo/discussions). Not really committed one way or another on that in this particular spot.

Reply:
I agree on the replacing forums and canny with GitHub discussions. We do not have a special label or category for forums to accept proposals so I guess directing them to GitHub is better. Would you also like the canny link to be removed?

Reply:
I think linking to canny and Github for now sounds good. We're still keeping Canny up-to-date at this point

Reply:
Done!